queries/okta_queries/okta_harfile_iocs.yml

```
@@ -0,0 +1,25 @@
+AnalysisType: saved_query
+QueryName: "Okta HAR File IOCs"
+Description: https://sec.okta.com/harfiles
+Query: |4
```

is this right? was expecting to see


```suggestion
Query: |-
```